Sources_v3/Controllers/MessageController/MessageController.swift

```diff
@@ -179,6 +179,19 @@ public extension _ChatMessageController {
             }
         }
     }
+    
+    /// Load replies for message this controller observes.
+    ///
+    /// - Parameters:
+    ///   - pageSize: Number of messages that should be fetched.
+    ///   - completion: The completion. Will be called on a **callbackQueue** when the network request is finished.
+    ///                 If request fails, the completion will be called with an error.
+    ///
+    func loadReplies(pageSize: Int = 25, completion: ((Error?) -> Void)? = nil) {
+        messageUpdater.loadReplies(cid: cid, messageId: messageId, pagination: .init(pageSize: pageSize)) { [weak self] error in
+            self?.callback { completion?(error) }
+        }
+    }
```

Comment:
We agreed on a different form of public API for `MessageController`. This shouldn't be here.

Reply:
Removed. 
Will be part of the next PR.